### PR TITLE
Bug 1373753 - XCUITestCase NoImageTests for Show and hide images

### PR DESCRIPTION
**Pull request by ugarlapati**

### CODE REVIEW

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
NoImageTests.swift | 56 | Limit vertical whitespace to a single empty line. Currently 2. 
NoImageTests.swift | 1 | Files should have consistent header comments. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: XCUITests/NoImageTests.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
+//
+//  NoImageTests.swift
+//  Client
+//
+//  Created by Usha Garlapati on 6/13/17.
+//  Copyright © 2017 Mozilla. All rights reserved.
+//
+
```
Comment (npark-mozilla) [inline XCUITests/NoImageTests.swift]: It should have the header as:

/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

File: XCUITests/NoImageTests.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        
+        // Load a same page on a new tab, check images are hidden
+        //navigator.goto(NewTabScreen)
+        navigator.openNewURL(urlString: url1)
+        
+        // Open it, then select show images it, and check it's showing the images
+        showImages()
+        checkHideImages()
+    }
+
```
Comment (npark-mozilla) [inline XCUITests/NoImageTests.swift]: remove this empty line

File: XCUITests/NoImageTests.swift [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        navigator.goto(BrowserTab)
+    }
+    
+    private func checkHideImages() {
+        navigator.goto(BrowserTabMenu)
+        app.collectionViews.containing(.cell, identifier:"FindInPageMenuItem").element.swipeLeft()
+        waitforExistence(app.collectionViews.cells["HideImageModeMenuItem"])
+        navigator.goto(BrowserTab)
+    }
+
```
Comment (npark-mozilla) [inline XCUITests/NoImageTests.swift]: remove this empty line

**Comment by npark-mozilla** [comment]
@ugarlapati looks good, you have to also squash the commits, and the commit message should be "Bug 1373753 - XCUITestCase NoImageTests for Show and hide images"

File: XCUITests/NoImageTests.swift [hunk 4]
```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+
```
Comment (npark-mozilla) [inline XCUITests/NoImageTests.swift]: @ugarlapati remove this empty space

**Pull request merged**